target/linux/ath79/dts/qca9533_mikrotik_routerboard-952ui-5ac2nd.dts

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-952ui-5ac2nd", "qca,qca9533";
+	model = "MikroTik hAP ac lite";
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_wan_pin>;
+
+		power {
+			label = "mikrotik:green:power";
```

Comment:
drop mikrotik: prefix for all LEDs, yielding e.g. "green:power" for this one etc.